magicai/Prompts.php

```diff
@@ -0,0 +1,295 @@
+	 * Prepare the input array for translating content to a specific language.
+	 *
+	 * @return array
+	 * @since 3.0.0
+	 */
+	public static function prepare_translation_input() {
+		$content        = Input::post( 'content', '' );
+		$language       = Input::post( 'language', '' );
+		$is_html        = Input::post( 'is_html', false, Input::TYPE_BOOL );
+		$system_content = 'Your task is to translate the provided text into {language}. Identify the original language if needed and ensure the translation accurately conveys the meaning of the original content.';
+		$system_content = str_replace( '{language}', $language, $system_content );
+
+		if ( ! $is_html ) {
+			$system_content .= ' Please respond with plain text only. Do not use markdown or HTML.';
+		}
+
+		return array(
+			'model'       => Models::GPT_4O,
+			'messages'    => array(
+				array(
+					'role'    => 'system',
+					'content' => $system_content,
+				),
+				array(
+					'role'    => 'user',
+					'content' => $content,
+				),
+			),
+			'temperature' => 0.7,
+		);
+	}
+
+	/**
+	 * Prepare the input array for rephrasing content
+	 *
+	 * @return array
+	 * @since 3.0.0
+	 */
+	public static function prepare_rephrase_input() {
+		$content        = Input::post( 'content', '' );
+		$is_html        = Input::post( 'is_html', false, Input::TYPE_BOOL );
+		$system_content = 'Your task is to rephrase any text content provided to you, ensuring that the original meaning is preserved while expressing it differently.';
+
+		if ( ! $is_html ) {
+			$system_content .= ' Please respond with plain text only. Do not use markdown or HTML.';
+		}
+
+		return array(
+			'model'       => Models::GPT_4O,
+			'messages'    => array(
+				array(
+					'role'    => 'system',
+					'content' => $system_content,
+				),
+				array(
+					'role'    => 'user',
+					'content' => $content,
+				),
+			),
+			'temperature' => 0.7,
+		);
+	}
+
+	/**
+	 * Prepare the input array for making the content shorten.
+	 *
+	 * @return array
+	 * @since 3.0.0
+	 */
+	public static function prepare_make_shorter_input() {
+		$content        = Input::post( 'content', '' );
+		$is_html        = Input::post( 'is_html', false, Input::TYPE_BOOL );
+		$system_content = 'Your task is to condense the provided text, retaining the key points and meaning while making the content as concise as possible.';
+
+		if ( ! $is_html ) {
+			$system_content .= ' Please respond with plain text only. Do not use markdown or HTML.';
+		}
+
+		return array(
+			'model'       => Models::GPT_4O,
+			'messages'    => array(
+				array(
+					'role'    => 'system',
+					'content' => $system_content,
+				),
+				array(
+					'role'    => 'user',
+					'content' => $content,
+				),
+			),
+			'temperature' => 0.7,
+		);
+	}
+
+	/**
+	 * Prepare the input array for changing the tone of the content.
+	 *
+	 * @return array
+	 * @since 3.0.0
+	 */
+	public static function prepare_change_tone_input() {
+		$content        = Input::post( 'content', '' );
```

Comment:
`Reusability`
We can make the method more loosely coupled by passing parameters directly, rather than depending on the request's POST data.